SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.cpp

```diff
@@ -216,3 +245,59 @@ void FSpatialLoadBalancingHandler::LogMigrationFailure(EActorMigrationResult Act
 		}
 	}
 }
+
+bool FSpatialLoadBalancingHandler::EvaluateRemoteMigrationComponent(const AActor* NetOwner, const AActor* TargetActor,
+																	VirtualWorkerId& WorkerId)
+{
+	if (TargetActor != nullptr)
+	{
+		if (TargetActor->HasAuthority())
+		{
+			UE_LOG(LogSpatialLoadBalancingHandler, Warning, TEXT("Should call AController::Possess"));
```

Comment:
This is the case where both the controller and the target are on the same worker. Can we then make the code in OnAuthorityGained happen anyway? (maybe on the next game frame because properties might be set on the component after creation).
It might make the user code simpler.